shared/p2p/hobbits/BUILD.bazel

```diff
@@ -0,0 +1,48 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "gossip_handler.go",
+        "hobbits.go",
+        "process.go",
+        "rpc_handler.go",
+        "server.go",
+    ],
+    importpath = "github.com/prysmaticlabs/prysm/shared/p2p/hobbits",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//beacon-chain/db:go_default_library",
+        "//proto/beacon/p2p/v1:go_default_library",
+        "//proto/eth/v1alpha1:go_default_library",
+        "//shared:go_default_library",
+        "//shared/event:go_default_library",
+        "//shared/hashutil:go_default_library",
+        "//shared/p2p:go_default_library",
+        "@com_github_gogo_protobuf//proto:go_default_library",
+        "@com_github_libp2p_go_libp2p_peer//:go_default_library",
+        "@com_github_pkg_errors//:go_default_library",
+        "@com_github_prysmaticlabs_go_ssz//:go_default_library",
+        "@com_github_renaynay_go_hobbits//encoding:go_default_library",
+        "@com_github_renaynay_go_hobbits//tcp:go_default_library",
+        "@com_github_renekroon_ttlcache//:go_default_library",
+        "@com_github_sirupsen_logrus//:go_default_library",
+        "@in_gopkg_mgo_v2//bson:go_default_library",
+    ],
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = ["server_test.go"],
+    embed = [":go_default_library"],
+    deps = [
+        "//beacon-chain/core/state:go_default_library",
```

Comment:
This is causing a visibility failure and that's our first red flag that we're doing something wrong by giving hobbits direct database access. Let's trim down this PR to remove the request handling then follow up with another PR to wire hobbits to the beacon-chain p2p request handlers (regular sync).

Reply:
+1